portfolio/src/main/webapp/blog.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <title>Blog - Neel Gandhi</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel=icon href=/images/favicon.ico>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:
+              wght@300&display=swap" rel="stylesheet">
```

Comment:
Should this be broken up into two lines where it is?

Reply:
I had split it up to not go over the 80 characters per line, but wasn't sure if that was necessary

Reply:
Ah, I see. I think it's the right thing to do, I just didn't realize that that was over 80 due to how it's being displayed.